### DISCUSSION TITLE
PieChartSectionData : Title position

### DISCUSSION THREAD

**Discussion by ASDdgomez** [imaNNeo/fl_chart#1137] [Q&A]
Hi,
Thanks for all.

My question:¿How can I move the titles from position when the percentages are. very small, attached graphic example ?

<img width="584" alt="image" src="https://user-images.githubusercontent.com/103059856/188485959-34edf1f2-9ff3-46b2-b5cc-663cee3965cb.png">

**Comment by imaNNeo** [comment]
We don't support this feature at this time. Please search in the issues list. If you didn't find any related issues, please file an issue.
